python/rascaline/examples/le-basis.py

```diff
@@ -0,0 +1,197 @@
+"""
+.. _example-le-basis:
+
+Laplacian eigenstate basis
+==========================
+
+.. start-body
+"""
+
+# %%
+#
+# This script illustrates how to generate a spherical expansion using
+# the Laplacian eigenstate basis (https://doi.org/10.1063/5.0124363),
+# both using truncation with l_max, n_max hyper-parameters and with an
+# eigenvalue threshold.
+
+import ase.io
+import matplotlib.pyplot as plt
+import numpy as np
+import scipy.optimize
+from metatensor import Labels, TensorBlock, TensorMap
+from scipy.special import jv
+
+import rascaline
+
+
+# %%
+#
+# First using a truncation with l_max, n_max hyper-parameters (easy):
+cutoff = 4.4
+l_max = 6
+n_max = 8
+
+structures = ase.io.read("dataset.xyz", ":10")
+
+spliner = rascaline.utils.SoapSpliner(
```

Comment:
A comment and link to the splined radial integral example could be useful.

Also cross reference this example from the splined radial integral example. Saying something along the lines that rascaline is also able to perform the radial integral numerically for some combinations of basis functions.

Reply:
> Saying something along the lines that rascaline is also able to perform the radial integral numerically for some combinations of basis functions.

I think we are still missing something like this, in general explaining what this splining code does and why